rules/base.js

```diff
@@ -38,6 +41,9 @@ module.exports = {
         'declaration-colon-newline-after': 'always-multi-line',
         'declaration-colon-space-after': 'always-single-line',
         'declaration-colon-space-before': 'never',
+        'declaration-property-value-blacklist': {
+            '/^(?!--).*(background|color|fill|stroke|border|shadow|outline).*/': ['/#([a-f0-9]{3,})*/i', '/(rgb|hsl)a?\\(/'],
```

Comment:
```suggestion
            '/^(?!--).*(background|color|fill|stroke|border|shadow|outline)/': ['/#([a-f0-9]{3,})*/i', '/(rgb|hsl)a?\\(/'],
```

Reply:
Also, do we need the `.*` after `(?!--)`?

Reply:
Yes, for cases like `text-decoration-color`